### PR TITLE
Features/improve gas aggregation

### PR DESCRIPTION
**Pull request by AmeliaNadal**
This PR has for aim to improve the aggregation of the gas stores and generators by:

- deleting the specific task that was previously doing it,
- including the aggregation directly in the specific tasks before the insertion of these components in the DB.

In this way, the aggregation of gas stores and generators will be consistent to the one of industrial gas loads (see https://github.com/openego/eGon-data/pull/761).

## Before merging into `dev`-branch, please make sure that

- [x] the `CHANGELOG.rst` was updated.
- [x] new and adjusted code is formated using `black` and `isort`.
- [x] the `Dataset`-version is updated when existing datasets are adjusted.
- [x] the branch was merged into the
      `continuous-integration/run-everything-over-the-weekend`-branch.
- [x] the workflow is running successful in `test mode`.
- [x] the workflow is running successful in `Everything` mode.

### CODE REVIEW

**Comment by CarlosEpia** [comment]
@AmeliaNadal I will be available again to check this PR on Wednesday 01.06. If it is not urgent, I can take care of it that day. Is it ok?

**Comment by AmeliaNadal** [comment]
> @AmeliaNadal I will be available again to check this PR on Wednesday 01.06. If it is not urgent, I can take care of it that day. Is it ok?

Yes, this is completely fine :)

**Review by CarlosEpia — COMMENTED** [review]
Codewise it looks ok. Please use black in all the modified files. Do you consider that I should run egon-data to test it, or the tests in your local machine and in the CI branch are enough?

File: CHANGELOG.rst [hunk 1]
```diff
@@ -357,6 +357,8 @@ Changed
   `#581 <https://github.com/openego/eGon-data/issues/581>`_
 * Update deposit id to access v0.7 of the zenodo repository
   `#736 <https://github.com/openego/eGon-data/issues/736>`_
+* Improve CH4 stores and productions aggregation by removing dedicated task
+  `#PR775 <https://github.com/openego/eGon-data/pull/775>`_
```
[the remark this answers is not in the document]
Comment (AmeliaNadal) [reply CHANGELOG.rst:361]: There is no issue associated to the PR, should I create one?

**Comment by AmeliaNadal** [comment]
> @AmeliaNadal could you send me the link of the issue where that motivated this PR, please?

There is no issue associated to the PR, should I create one?

**Comment by AmeliaNadal** [comment]
> Codewise it looks ok. Please use black in all the modified files. Do you consider that I should run egon-data to test it, or the tests in your local machine and in the CI branch are enough?

I applied black and isort to all the modified files (https://github.com/openego/eGon-data/pull/775/commits/dc35548ada33da4fe3e2b8bf95de132fc5c53238) and I don't think you need to run this branch to test it, but it is up to you if you think it is neccessary to approve the PR ;)

**Comment by CarlosEpia** [comment]
> > @AmeliaNadal could you send me the link of the issue where that motivated this PR, please?
> 
> There is no issue associated to the PR, should I create one?

It is mostly to give a bit of context of the purpose of the changes. In this case, I think we can omit it.

**Comment by CarlosEpia** [comment]
> > Codewise it looks ok. Please use black in all the modified files. Do you consider that I should run egon-data to test it, or the tests in your local machine and in the CI branch are enough?
> 
> I applied black and isort to all the modified files ([dc35548](https://github.com/openego/eGon-data/commit/dc35548ada33da4fe3e2b8bf95de132fc5c53238)) and I don't think you need to run this branch to test it, but it is up to you if you think it is neccessary to approve the PR ;)

Since the changes are not so big, the changes are straightforward, and it was tested for SH and Germany I don't consider it necessary to run this branch.

**Review by CarlosEpia — APPROVED** [review]
This branch can be merged.

**Comment by AmeliaNadal** [comment]
Thanks for your review @CarlosEpia, I've add some context details in the introduction of the PR (sorry for having done that so lately).

**Pull request merged**